src/container.c

```diff
@@ -465,7 +465,7 @@ static int hyper_rescan_scsi(void)
 	num = scandir("/sys/class/scsi_host/", &list, NULL, NULL);
 	if (num < 0) {
 		perror("scan /sys/class/scsi_host/ failed");
-		return -1;
+		return -ENOENT;
```

Comment:
I'm ok to move `rescan` to the case that hyperstart uses scsi device as container root device, 
```
if (container->fstype) {
    hyper_rescan_scsi();
}
```